templates/.github/workflows/js-test-and-release.yml

```diff
@@ -0,0 +1,125 @@
+name: test & maybe release
+on:
+  push:
+    branches:
+      - $default-branch # with #262 - ${{{ github.default_branch }}}
+  pull_request:
+    branches:
+      - $default-branch # with #262 - ${{{ github.default_branch }}}
+
+jobs:
+
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: lts/*
+    - uses: ipfs/aegir/actions/cache-node-modules@master
```

Comment:
This action doesn't have to live in the `aegir` repo - when we're happy with the workflow it could be moved here.

Reply:
We could also make a separate repo for it if we want to put it on the marketplace - similarly to what we did here https://github.com/protocol/cache-go-action. For now though, we can leave it in `ipfs/aegir`.